testing/environments/docker/kibana/config/kibana-x-pack.yml

```diff
@@ -6,4 +6,12 @@ server.host: "0"
 elasticsearch.url: http://elasticsearch:9200
 elasticsearch.username: elastic
 elasticsearch.password: changeme
-xpack.monitoring.ui.container.elasticsearch.enabled: true
+#xpack.monitoring.ui.container.elasticsearch.enabled: true
```

Comment:
@jbudz This config option is in the new default docker file from Infra. It seems when I have this line in on startup it rebuilds kibana again. I read this about the plugins enabling below. Is this also the case for this var?

Reply:
I wouldn't expect this to, the rough heuristic is anything that changes the client side code will cause a rebundle.

I tried toggling this on the docker.elastic.co/kibana/kibana:6.1.1 images and it didn't cause any rebuilds - I'll give it a try with your dockerfile.

Reply:
The image used here is almost identical to our official image with x-pack based on 7.0.0-alpha1-SNAPSHOT. I assumed we the exact same build is used for our filebeat and libbeat build, but I didn't see it on both builds (also 1 passed). So either it could be that I played around with too many different things and things got messed up when I tried to reproduce it or it's not always triggered. 

Are there any environment variables that can trigger a rebuild?